software/donetick.yml

```
@@ -0,0 +1,17 @@
+name: "Donetick"
+website_url: "https://donetick.com"
+source_code_url: "https://github.com/donetick/donetick"
+description: "Task and chore management tool for personal and family use. with advanced scheduling, flexible assignment, and group sharing capabilities, detailed history, automation via API. with simple and modern design."
```

Please make the description under 250 characters long.
```suggestion
description: "Task and chore management tool for personal and family use. with advanced scheduling, flexible assignment, and group sharing capabilities, detailed history, automation via API. with simple and modern design."
```

---

Hi @Rabenherz112 ! I try a few character counters and see this as 207 characters. also, the suggested change is exactly the same

---

Nevermind, I accidentally copied your description 2x in the word counter.

---

That's cheating :)